x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/es_search_source.js

```diff
@@ -241,10 +241,17 @@ export class ESSearchSource extends AbstractESSource {
 
     const searchSource = await this._makeSearchSource(searchFilters, 0);
     searchSource.setField('aggs', {
+      totalEntities: {
+        cardinality: {
+          field: topHitsSplitField,
+          precision_threshold: 10,
```

Comment:
not sure why we motivate lowering this from the default. Is this something the maps-app should be concerned about?

Reply:
Just saves memory in the Elasticsearch side. Since we don't care about precision, lets avoid ES using memory for more precise counts. I could lower to 1 if that would be clearer.

Reply:
+1

Reply:
+1 on explanation or +1 to change precision_threshold to 1?

Reply:
changing to 1 😂